.gitignore

```diff
@@ -15,3 +15,4 @@ MANIFEST
 # Used in demo apps
 secrets.cfg
 .mypy_cache/
+/.idea
```

Comment:
The repo's `.gitignore` file should only contain files whose existence is caused by something in the repo. Files created by a user's editor should be in that user's `core.excludesfile`.